### PR TITLE
Add `Stringified` type

### PR DESCRIPTION
**Pull request by youssefsharief**
I believe this type can have a lot of uses.

### CODE REVIEW

**Comment by sindresorhus** [comment]
What uses? Preferably link to some real code using it.

**Comment by youssefsharief** [comment]
I used before to stringify my interface so that I am able to use the object as a form model. Let us say I have a car with the speed prop. This prop is a number in the domain model, but when I want to use this in a form, I need to stringify it.

File: source/stringified.d.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+@example
+```
+import {Stringified} from 'type-fest';
+
+type Foo = {
+	a: number;
+	b: string;
+};
+
+type FooWithoutA = Stringified<Foo, 'a'>;
```
Comment (bfred-it) [inline source/stringified.d.ts]: `Stringified` only has one generic argument: `ObjectType`

What does `'a'` map to?

Comment (youssefsharief) [reply source/stringified.d.ts]: Yes, I was wrong on that. I fixed it

**Comment by sindresorhus** [comment]
Makes sense. Can you please follow each step in https://github.com/sindresorhus/type-fest/blob/master/.github/contributing.md

File: readme.md [hunk 2]
```diff
@@ -71,6 +71,7 @@ Click the type names for complete docs.
 - [`RequireAtLeastOne`](source/require-at-least-one.d.ts) - Create a type that requires at least one of the given properties.
 - [`ReadonlyDeep`](source/readonly-deep.d.ts) - Create a deeply immutable version of a `object`/`Map`/`Set`/`Array` type.
 - [`LiteralUnion`](source/literal-union.d.ts) - Create a union type by combining primitive types and literal types without sacrificing auto-completion in IDEs for the literal type part of the union. Workaround for [Microsoft/TypeScript#29729](https://github.com/Microsoft/TypeScript/issues/29729).
+- [`Stringified`](source/stringified.d.ts) - Create a stringified type of an object
```
Comment (sindresorhus) [inline readme.md]: Move it to the bottom (same in the other places)

File: readme.md [hunk 3]
```diff
@@ -71,6 +71,7 @@ Click the type names for complete docs.
 - [`RequireAtLeastOne`](source/require-at-least-one.d.ts) - Create a type that requires at least one of the given properties.
 - [`ReadonlyDeep`](source/readonly-deep.d.ts) - Create a deeply immutable version of a `object`/`Map`/`Set`/`Array` type.
 - [`LiteralUnion`](source/literal-union.d.ts) - Create a union type by combining primitive types and literal types without sacrificing auto-completion in IDEs for the literal type part of the union. Workaround for [Microsoft/TypeScript#29729](https://github.com/Microsoft/TypeScript/issues/29729).
+- [`Stringified`](source/stringified.d.ts) - Create a stringified type of an object.
```
Comment (sindresorhus) [inline readme.md]: The description is not clear enough. It doesn't stringify the object itself.

File: source/stringified.d.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+type Foo = {
+	a: number;
+	b: string;
+};
+
+type StringifiedFoo = Stringified<Foo>;
+//=> {
+	a: string;
+	b: string;
+};
```
Comment (sindresorhus) [inline source/stringified.d.ts]: Use multiline comment.

File: readme.md (hunk 3 above)
Comment (sindresorhus) [reply readme.md]: This was not resolved. It's still not clear enough what it does.

File: source/stringified.d.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
+/**
+Create a stringified type of an object.
```
Comment (sindresorhus) [inline source/stringified.d.ts]: Please spend some more time on the description.

Also per PR guidelines, you need to include some use-cases in text form.

File: source/stringified.d.ts (hunk 4 above)
Comment (sindresorhus) [reply source/stringified.d.ts]: Don't resolve things that are not resolved.

**Comment by sindresorhus** [comment]
`Stringified` makes me think it converts the whole object to a string or something. Maybe we should be explicit and call it `StringifiedKeys`? Any other naming ideas?

**Comment by sindresorhus** [comment]
@youssefsharief Friendly bump :)

**Comment by sindresorhus** [comment]
Closing for lack of response: https://github.com/sindresorhus/type-fest/issues/87

**Pull request closed without merging**